# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/pcmcia-cs/rc.pcmcia.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./etc/install-etc.orig	2003-01-18 08:38:22.000000000 +0100
+++ ./etc/install-etc	2003-01-18 12:00:46.000000000 +0100
@@ -50,6 +50,14 @@
 
 install_sysv ()
 {
+	cp rc.pcmcia /etc/rc.d/init.d/pcmcia
+	chmod +x /etc/rc.d/init.d/pcmcia
+	echo "The script 'pcmcia' has the priority 04." \
+		> /etc/rc.d/rcX.d/X04pcmcia
+}
+
+install_sysv_orig ()
+{
     SYSV=$PREFIX$RC_DIR/init.d/pcmcia
 
      if [ -d /etc/sysconfig ] ; then
--- ./etc/rc.pcmcia.orig	2002-08-08 08:43:43.000000000 +0200
+++ ./etc/rc.pcmcia	2003-01-18 08:38:22.000000000 +0100
@@ -1,4 +1,7 @@
 #!/bin/sh
+#
+# Desc: loading the PCMCIA drivers
+#
 
 # rc.pcmcia 1.45 2002/08/08 06:43:43 (David Hinds)
 #
@@ -41,7 +44,7 @@
     # Put pcmcia_core options here
     CORE_OPTS=
     # Put cardmgr options here
-    CARDMGR_OPTS=
+    CARDMGR_OPTS=-f
     # To set the PCMCIA scheme at startup...
     SCHEME=
 fi
